.gitlab-ci.yml

```diff
@@ -52,7 +52,9 @@ before_script:
   - eval $(opam config env)
   - opam config list
   - opam install -j ${NJOBS} -y camlp5.${CAMLP5_VER} ocamlfind num ${EXTRA_OPAM}
-  - rm -rf ~/.opam/log/
+  - rm -rf ~/.opam/log/*
+  - rm -rf ~/.opam/archives/*
+  - rm -rf ~/.opam/${COMPILER}/build/*
```

Comment:
What's this about?

Reply:
I was trying to improve caching time, now I see that .opam contains all 4 compilers.

Reply:
Anyways this cleaning is convenient to do, but I'll move it to a separate commit.